src/App.js

```diff
@@ -1,12 +1,65 @@
-import React from 'react';
-import './App.css';
+import React, {useState, useEffect} from 'react';
+import axios from 'axios';
+import Pokemon from './components/pokemon/Pokemon'
+import Button from './components/button/Button'
+import logo from './assets/logo.png';
+import './App.css'
 
 function App() {
-  return (
-    <div>
-      Begin hier met de opdracht!
-    </div>
-  );
+    const [pokemons, setPokemons] = useState([]);
```

Comment:
Als `pokemon` een object zal worden, dan zou ik een default waarde van `undefined` pakken. Nu lijkt het alsof het een array is en zal blijven, maar dat is niet zo
```suggestion
    const [pokemons, setPokemons] = useState();
```